api/cmd/main.go

```diff
@@ -176,9 +177,9 @@ func main() {
 	}
 	tracker.Start()
 
+	mlflowClientFactory := mlflow.NewClientFactory()
```

Comment:
Refer to this https://github.com/gojek/merlin/pull/84/files#r582762090 ,
It's better to instantiate mlflow client directly here.

Reply:
i have addressed this